api/src/main/resources/workbench.yaml

```diff
@@ -141,38 +141,61 @@ paths:
         "204":
           description: Account deleted successfully.
 
-  /api/v1/register:
+  /api/v1/id-verification:
     post:
       tags:
         - profile
-      description: Registers a profile for the user in AllOfUs and Firecloud; called after the user has consented to the terms of use
-      operationId: register
+      description: Accepts identity information for verification.
```

Comment:
How do these three methods relate to one another? Do they need to be called in a particular order? Does something happen once all of them have been called?

Reply:
At the moment I'm assuming they could be called in any order -- the user chooses which steps to do when.

Once the user has completed ID verification, TOS, ethics training, and the demographic survey, their data access level switches to REGISTERED, and (this isn't implemented yet) we add them to the authorization domain for access to the CDR.

Reply:
agreed we want to allow these to occur in any order! (The UI may guide the user toward some specific order...)